fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalBiomeTags.java

```diff
@@ -0,0 +1,170 @@
+	public static final TagKey<Biome> IS_JUNGLE = register("is_jungle");
+	/**
+	 * For temperate or warmer plains-like biomes.
+	 * For snowy plains-like biomes, see {@link ConventionalBiomeTags#IS_SNOWY_PLAINS}.
+	 */
+	public static final TagKey<Biome> IS_PLAINS = register("is_plains");
+	public static final TagKey<Biome> IS_SAVANNA = register("is_savanna");
+	/**
+	 * For land biomes where ice naturally spawns.
+	 * For biomes where snow alone spawns, see {@link ConventionalBiomeTags#IS_SNOWY}.
+	 */
+	public static final TagKey<Biome> IS_ICY = register("is_icy");
+	/**
+	 * For water biomes where ice naturally spawns.
+	 * For biomes where snow alone spawns, see {@link ConventionalBiomeTags#IS_SNOWY}.
+	 */
+	public static final TagKey<Biome> IS_AQUATIC_ICY = register("is_aquatic_icy");
+	/**
+	 * Biomes that are dedicated to spawning on the shoreline of a body of water.
+	 */
+	public static final TagKey<Biome> IS_BEACH = register("is_beach");
+	/**
+	 * Biomes densely populated with deciduous trees.
+	 */
+	public static final TagKey<Biome> IS_FOREST = register("is_forest");
+	public static final TagKey<Biome> IS_BIRCH_FOREST = register("is_birch_forest");
+	public static final TagKey<Biome> IS_OCEAN = register("is_ocean");
+	public static final TagKey<Biome> IS_DESERT = register("is_desert");
+	public static final TagKey<Biome> IS_RIVER = register("is_river");
+	public static final TagKey<Biome> IS_SWAMP = register("is_swamp");
+	public static final TagKey<Biome> IS_MUSHROOM = register("is_mushroom");
+	public static final TagKey<Biome> IS_UNDERGROUND = register("is_underground");
+	public static final TagKey<Biome> IS_MOUNTAIN = register("is_mountain");
+
+	public static final TagKey<Biome> IS_HOT = register("is_hot");
+	public static final TagKey<Biome> IS_HOT_OVERWORLD = register("is_hot/overworld");
+
+	public static final TagKey<Biome> IS_TEMPERATE = register("is_temperate");
+	public static final TagKey<Biome> IS_TEMPERATE_OVERWORLD = register("is_temperate/overworld");
+
+	public static final TagKey<Biome> IS_COLD = register("is_cold");
+	public static final TagKey<Biome> IS_COLD_OVERWORLD = register("is_cold/overworld");
+
+	public static final TagKey<Biome> IS_WET = register("is_wet");
+	public static final TagKey<Biome> IS_WET_OVERWORLD = register("is_wet/overworld");
+
+	public static final TagKey<Biome> IS_DRY = register("is_dry");
+	public static final TagKey<Biome> IS_DRY_OVERWORLD = register("is_dry/overworld");
+
+	public static final TagKey<Biome> IS_VEGETATION_SPARSE = register("is_sparse_vegetation");
+	public static final TagKey<Biome> IS_VEGETATION_SPARSE_OVERWORLD = register("is_sparse_vegetation/overworld");
+
+	public static final TagKey<Biome> IS_VEGETATION_DENSE = register("is_dense_vegetation");
+	public static final TagKey<Biome> IS_VEGETATION_DENSE_OVERWORLD = register("is_dense_vegetation/overworld");
+
+	public static final TagKey<Biome> IS_CONIFEROUS_TREE = register("is_tree/coniferous");
```

Comment:
Question: How does `is_tree` relate to biomes / the above `is_forest`

Reply:
`is_forest` is the copy of the vanilla `minecraft:is_forest` which includes not just forest, dark forest, and birch forest but it also includes grove which is not deciduous. It does not contain other taiga biomes. Fun mojang moment lol
`
Some people want to target specific biomes based on the kind of trees it has so `is_tree/coniferous` contains both the `is_taiga` biome tag and grove biome so you get all the real spruce trees biomes since vanilla's tags are wack. And `is_tree/deciduous` gives you all the real biomes that aren't savanna, jungle, or spruce.

A use case can be you have a beetle mob that attacks only Jungle trees. The Jungle trees could be in non-jungle biomes as a sorta tree mix biome. The jungle tree could be placed but a custom configured/placedfeature too so can't scan for the feature. So you would use `is_tree/jungle` biome tag to add the beetle to the biomes that have jungle trees since the wood type is your target. Not the kind of biome. (This was roughly the kind of use case iirc that people asked for is_tree/ tags)

Reply:
I see, somewhat of a niche usecase but at least it has one 👍